techdocs/docs/using-the-package/apis-&-components/modules/config.md

```diff
@@ -0,0 +1,106 @@
+# configModule
+
+The `configModule` function provides a `/config` endpoint used to pass configuration variables to the frontend.
+
+!!! warning "Warning"
+    DO NOT send secrets or confidential data through this endpoint. It is not secure to pass these along to the frontend and you should find another solution that doesnt involve sending or storing secrets in the frontend.
+
+## Import
+
+```JavaScript
+// ESModule Syntax (preferred)
+import { configModule } from "@bcgov/citz-imb-express-utilities";
+
+// CommonJS Syntax
+const { configModule } = require('@bcgov/citz-imb-express-utilities');
+```
+
+## Usage
+
+A basic example of using the `configModule` function.
+
+```JavaScript
+import express from 'express';
+import { configModule } from '@bcgov/citz-imb-express-utilities';
+
+// Define Express App
+const app = express();
+
+configModule(app);
+```
+
+A basic example of calling the config route in the frontend:
+
+```TypeScript
+// src/globals.ts
+
+// Allow Window.configuration from config endpoint.
+interface Window {
+  configuration: Record<string, unknown>;
+}
+```
+
+```TypeScript
+// src/App.tsx
+
+// Fetchs config data and sets it on the window object 
+// so it can be accessed from anywhere in the application using window.configuration.
+export const App = () => {
+  useEffect(() => {
+    (async () => {
+      const response = await fetch('/api/health');
+      const config = await response.data();
+      window.configuration = config;
+    });
+  }, []);
+
+  return (
+    // ...
+  )
+}
+```
+
+## TypeScript Type
+
+<!-- The following code block is auto generated when types in the package change. -->
+<!-- TYPE: configModule -->
+```TypeScript
+// placeholder
```

Comment:
is this supposed to be placeholder for now

Reply:
Yeah the auto documenting types will update this